torch_geometric/explain/algorithm/base.py

```diff
@@ -43,14 +45,17 @@ def forward(
                 in a multi-task learning scenario. Should be kept to
                 :obj:`None` in case the model only returns a single output
                 tensor. (default: :obj:`None`)
+            is_hetero (bool, optional): Whether the input graph is
+                heterogeneous. (default: :obj:`False`)
```

Comment:
We can detect if its Heterogenous if `x` is a dict we don't need the `is_hetero` arg.